### DISCUSSION TITLE
How to Multi Proc Cmd?

### DISCUSSION THREAD

**Discussion by miasik** [bbc-mc/sdweb-merge-block-weighted-gui#33] [Q&A]
How to use the "Multi Proc Cmd" window?

**Comment by CrazylegsPDX** [comment]
There is a read me file in extensions > sdweb-merge-block-weighted-gui called "README_each" that can be opened with notepad or text editor, with some notes there that roughly explain how to use it.

What I was able to piece together:

- **O="insert name here"** is the name you give your output file. Follow this with a comma.
- I recommend setting the Preset weight to ALL_A -- this will make all blocks not specific 100% model A
- **IN_A_##=#.#** specifies the number of the IN block you are targeting i.e., IN_A_03, and the value after the = specifies what % of the block should remain A. i.e., IN_A_03=.7 the IN03 block for A will absorb 30% of model B
- Not sure whether it is required to specify the corresponding B block, but I do and it works.
- The readme mentions specifying the A model and B model in your command -- this isn't necessary.

**Example multi proc command**
Below tests 10% increment adjustments to Model A & B IN05-IN08 and M blocks to test how the blocks effect Posing of figures.
- 10A/90B
- 20A/80B
- 30A/70B
- 40A/60B
- 50A/50B
- 60A/40B

O=Test-IN05-08_A10B90, IN_A_00=1, IN_B_00=0, IN_A_01=1, IN_B_01=0, IN_A_02=1, IN_B_02=0, IN_A_03=1, IN_B_03=0, IN_A_04=1, IN_B_04=0, IN_A_05=0.1, IN_B_05=0.9, IN_A_06=0.1, IN_B_06=0.9, IN_A_07=0.1, IN_B_07=0.9, IN_A_08=0.1, IN_B_08=0.9, M_A_00=0.1, M_B_00=0.9

O=Test-IN05-08_A20B80, IN_A_00=1, IN_B_00=0, IN_A_01=1, IN_B_01=0, IN_A_02=1, IN_B_02=0, IN_A_03=1, IN_B_03=0, IN_A_04=1, IN_B_04=0, IN_A_05=0.2, IN_B_05=0.8, IN_A_06=0.2, IN_B_06=0.8, IN_A_07=0.2, IN_B_07=0.8, IN_A_08=0.2, IN_B_08=0.8, M_A_00=0.2, M_B_00=0.8

O=Test-IN05-08_A30B70, IN_A_00=1, IN_B_00=0, IN_A_01=1, IN_B_01=0, IN_A_02=1, IN_B_02=0, IN_A_03=1, IN_B_03=0, IN_A_04=1, IN_B_04=0, IN_A_05=0.3, IN_B_05=0.7, IN_A_06=0.3, IN_B_06=0.7, IN_A_07=0.3, IN_B_07=0.7, IN_A_08=0.3, IN_B_08=0.7, M_A_00=0.3, M_B_00=0.7

O=Test-IN05-08_A40B60, IN_A_00=1, IN_B_00=0, IN_A_01=1, IN_B_01=0, IN_A_02=1, IN_B_02=0, IN_A_03=1, IN_B_03=0, IN_A_04=1, IN_B_04=0, IN_A_05=0.4, IN_B_05=0.6, IN_A_06=0.4, IN_B_06=0.6, IN_A_07=0.4, IN_B_07=0.6, IN_A_08=0.4, IN_B_08=0.6, M_A_00=0.4, M_B_00=0.6

O=Test-IN05-08_A50B50, IN_A_00=1, IN_B_00=0, IN_A_01=1, IN_B_01=0, IN_A_02=1, IN_B_02=0, IN_A_03=1, IN_B_03=0, IN_A_04=1, IN_B_04=0, IN_A_05=0.5, IN_B_05=0.5, IN_A_06=0.5, IN_B_06=0.5, IN_A_07=0.5, IN_B_07=0.5, IN_A_08=0.5, IN_B_08=0.5, M_A_00=0.5, M_B_00=0.5

O=Test-IN05-08_A60B40, IN_A_00=1, IN_B_00=0, IN_A_01=1, IN_B_01=0, IN_A_02=1, IN_B_02=0, IN_A_03=1, IN_B_03=0, IN_A_04=1, IN_B_04=0, IN_A_05=0.6, IN_B_05=0.4, IN_A_06=0.6, IN_B_06=0.4, IN_A_07=0.6, IN_B_07=0.4, IN_A_08=0.6, IN_B_08=0.4, M_A_00=0.6, M_B_00=0.4
